linux/just_files/just_entrypoint.sh

```diff
@@ -88,6 +88,15 @@ set -eu
 : ${VSI_COMMON_DIR=/vsi}
 : ${DOCKER_USERNAME=user}
 
+# In the case where docker_link_mounts causes /src and /vsi to be replaces with
+# symlinks, /vsi is needed for the initial entrypoint, but is replaces mid way
+# through. Which this is not really an issue, the pwd of the PID becomes stale
+# and warnings pop up:
```

Comment:
```suggestion
# In the case where docker_link_mounts causes /src and /vsi to be replaced with
# symlinks, /vsi is still needed for the initial entrypoint, but is replaced mid-way
# through. While this is not really an issue, the pwd of the PID becomes stale
# and warnings pop up:
```

Reply:
what do you mean it gets replaced mid-way through (that part of the sentance might be redundant and can just be deleted)